server/graphql/v2/mutation/VirtualCardMutations.ts

```diff
@@ -203,10 +206,26 @@ const virtualCardMutations = {
         UserId = user.id;
       }
 
-      return await virtualCard.update({
-        privateData: { ...(virtualCard.get('privateData') as Object), cvv, expireDate },
+      const updateAttributes = {
+        name: args.name,
         UserId,
-      });
+      };
```

Comment:
What if we don't pass an `args.assignee`? Will `UserId` be null? That's not desirable.

Same for `name`.